commands/fetch_templates.go

```diff
@@ -116,3 +116,27 @@ func moveTemplates(repoPath string, overwrite bool) ([]string, []string, error)
 
 	return existingLanguages, fetchedLanguages, nil
 }
+
+func pullTemplate(repository string) error {
```

Comment:
@martindekov  If you want to be able to filter to a specific template, I think you need to pass the name here and then pass it further down so that only that one template is copied.  Alternatively, we can remove the name filter for now and revisit it in the future.

Reply:
Yeah, will do that, will remember in general using variables in global scope is a bad thing, just thought as we are doing those things in the cli (yamlFIle) it is not mandatory :+1:

Reply:
On a second thought, as we cleared out I ment the repository, not a single template. Though the Idea is interesting and I would not resolve the conversation, we can open an Issue :100: